web/src/App.tsx

```diff
@@ -46,6 +49,14 @@ export default function App(): ReactElement {
   const shouldUseDarkMode = useDarkMode();
   const { t, i18n } = useTranslation();
   const canonicalUrl = useGetCanonicalUrl();
+  const setConsumptionAtom = useSetAtom(productionConsumptionAtom);
```

Comment:
What's the plan with this atom? It doesn't seem like we're using it anywhere

Reply:
It's used here `setConsumptionAtom(Mode.CONSUMPTION);` just below. 

It's here so if a user has production mode stored in local storage and we change the feature flag it will set this to production

Reply:
Ahh gotcha gotcha 👍